.github/workflows/build-targets.yml

```
@@ -0,0 +1,52 @@
+name: Build ML examples
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+  schedule:
+    - cron: '30 22 28 * 0'
```

This would only run by schedule if day 28 of a month happens to be a Sunday.
```suggestion
    - cron: '30 22 * * 0'
```